### DISCUSSION TITLE
[FR] Android 8.0 / SDK 26 support (why not?)

### DISCUSSION THREAD

**Discussion by eMPee584** [LSPosed/LSPosed#670] [Q&A] [answered]
Why was it removed, and how complicated is it to bring it back? What in d1867122d0a91de073c9fc1755a8bf8bb9e09f32 prevents a setting back `androidMinSdkVersion=26` ?
There are people stuck on 8.0 OEM ROMs, so the advice "(please at least use 8.1 because Android 8 is buggy)" from the changelog is a bit hard to follow for us..

**Comment by yujincheng08** [comment, marked as answer]
We enabled optimization and shrinking in d186712, which requires us to load modules by `DelegateLastClassLoader`, which does not exist before SDK 27.

**Reply by eMPee584** [reply]
Okey thanks for the explanation. It's not feasible to create any builds without that optimization, right?
